website/src/css/custom.css

```
@@ -66,3 +66,47 @@
     font-size: 1.2rem !important;
   }
 }
+
+/* Animations */
+@keyframes fadeInUp {
```

Stylelint (17.13.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)